apps/src/feedback.js

```diff
@@ -748,6 +748,35 @@ exports.showGeneratedCode = function(Dialog) {
   dialog.show();
 };
 
+exports.showToggleBlocksError = function(Dialog) {
```

Comment:
Function params should be lower/camel case, maybe studioDialog to avoid name collision?